target/linux/mediatek/image/Makefile

```diff
@@ -21,21 +21,17 @@ endef
 # default all platform image(fit) build 
 define Device/Default
   PROFILES = Default $$(DEVICE_NAME)
-  KERNEL_NAME := zImage
+  KERNEL_NAME := Image
+  KERNEL = kernel-bin | lzma | \
+	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb
+  KERNEL_INITRAMFS = kernel-bin | lzma | \
+	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb
```

Comment:
Hmm, as it appears, one could just use
`KERNEL_INITRAMFS = $(KERNEL)`
here and get rid of some additional lines ...